assets/js/components/BatchEdit/Confirmation.jsx

```diff
@@ -171,24 +178,37 @@ const BatchEditConfirmation = ({
           )}
 
           {hasDataToPost ? (
-            <div className="notification is-danger is-light has-text-centered content">
-              <p>
-                <FontAwesomeIcon icon="exclamation-triangle" /> NOTE: This batch
-                edit will affect {numberOfResults} works. To execute this
-                change, type "I understand"
-              </p>
-
-              <div css={verifyInputWrapper}>
-                <UIFormInput
-                  onChange={handleConfirmationChange}
-                  name="confirmationText"
-                  label="Confirmation Text"
-                  placeholder="I understand"
-                  required
-                  data-testid="input-confirmation-text"
-                />
+            <React.Fragment>
+              <div className="column is-two-thirds">
+                <UIFormField label="Enter a name for this Batch Edit (optional)">
+                  <UIFormInput
+                    onChange={handleBatchNickname}
```

Comment:
Just for consistent naming, can we change this function name to `handleNicknameChange`?